src/StickPortal.js

```diff
@@ -192,9 +192,33 @@ function calculateLeft(
     return result - hostLeft
   }
 
+  if (scrollHost) {
```

Comment:
I guess we need the same extra handling for `calculateTop` and horizontal scrolling, too.

Reply:
Interestingly that doesn't seem to be the case. Look at the regression example at the bottom of the example page. Vertical scroll seems to "just work".